src/location.rs

```diff
@@ -540,7 +540,7 @@ pub fn save(
 #[allow(non_snake_case)]
 pub fn job_do_DC_JOB_MAYBE_SEND_LOCATIONS(context: &Context, _job: &Job) {
     let now = time();
-    let mut continue_streaming: libc::c_int = 1;
+    let mut continue_streaming = false;
```

Comment:
good catch! in fact, this bad default seems to be the reason, location streaming was never really disabled once started :)